frontend/src/app/main/ui/workspace/libraries.scss

```diff
@@ -0,0 +1,133 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@import "refactor/common-refactor.scss";
+.modal-overlay {
+  @include flexCenter;
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 1000;
```

Comment:
mmm maybe we can define some z-index variable instead of just a `1000` ??

Reply:
jajjaja this seam like a "border: red test". I'll check it